### PR TITLE
Checkout/form style

### PR DESCRIPTION
**Pull request by rylanmindable**
Updates to the CSS/HTML

### CODE REVIEW

**Comment by gaurav-bains** [comment]
Removing myself from Reviewer list as me and @rylanmindable both have made changes to the PR
@pjdarch please review this PR

**Review by pjdarch — COMMENTED** [review]
@gaurav-bains I know you've been playing with the input types here. Perhaps `inputmode` can help achieve the intended result. https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/ Stripe uses it in their form validation https://checkout.stripe.dev/preview

File: checkout/deafult.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
```
Comment (pjdarch) [inline checkout/deafult.css:1]: CSS file is misspelled.

**Review by pjdarch — COMMENTED** [review]
@rylan the Email ID field is a disabled field, but that's not easily identifiable from a customer perspective. I always try and click on it. Can you add a different background color?  Additionally I notice you set a width of 600px for `#aa-checkout-div .summary`. This prevents the summary box from extending past that width. Did you want to specify a maximum width for the checkout form as well?

@gaurav-bains I don't particularly like the wording of `Email ID`. From a customer point-of-view (considering our general demographic) I'd be confused by it. Can you update the label and validation error `'Error with retrieving Email ID, please contact Customer Service'` to just be Email?

File: checkout/deafult.css (hunk 1 above)
Comment (gaurav-bains) [reply checkout/deafult.css:1]: @pjdarch can you please elaborate?
or maybe update it using suggestion

File: checkout/components/checkoutForm.js [hunk 2]
```diff
@@ -20,14 +20,27 @@ app.component('checkout-form',{
...
+          The entire transaction will take place on a
+          secure server using SSL technology.
+        </p>
+        <div>
+          <img src="https://mindable.github.io/unbounce-scripts/assets/mcafee_badge.png" alt="McAfee">&nbsp; <img src="https://mindable.github.io/unbounce-scripts/assets/truste_badge.png" alt="TRUSTe">
+        </div>
+        <p class="questions">
+          <strong>Questions?</strong> Call Toll Free: 1-866-329-7640
+        </p>
+      </div>`,
```
[the remark this answers is not in the document]
Comment (gaurav-bains) [reply checkout/components/checkoutForm.js:43]: Thanks @pjdarch for catching that, this is now fixed

**Comment by gaurav-bains** [comment]
> @gaurav-bains I don't particularly like the wording of `Email ID`. From a customer point-of-view (considering our general demographic) I'd be confused by it. Can you update the label and validation error `'Error with retrieving Email ID, please contact Customer Service'` to just be Email?

I have changed the wording to say `Email` now

File: checkout/deafult.css (hunk 1 above)
Comment (gaurav-bains) [reply checkout/deafult.css:1]: I think I got it now, renamed file from `deafult.css` to `default.css`

**Comment by gaurav-bains** [comment]
> @gaurav-bains I know you've been playing with the input types here. Perhaps `inputmode` can help achieve the intended result. https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/ Stripe uses it in their form validation https://checkout.stripe.dev/preview

Thanks for article @pjdarch 
I was looking into this situation and thought that it could be separate exercise as there is no standard we can follow to work on all browsers, even if we ignore IE we have Desktop Safari to consider which is becoming a bottleneck. Since this needs more time and effort, I am thinking of creating an issue and tackling it later. What do you think?

File: checkout/deafult.css (hunk 1 above)
Comment (pjdarch) [reply checkout/deafult.css]: Thanks. I was not sure how to comment on the file itself.

**Comment by pjdarch** [comment]
> I was looking into this situation and thought that it could be separate exercise as there is no standard we can follow to work on all browsers, even if we ignore IE we have Desktop Safari to consider which is becoming a bottleneck. Since this needs more time and effort, I am thinking of creating an issue and tackling it later. What do you think?

No problem. We can do it at a later date.

**Comment by rylanmindable** [comment]
I added a disabled cursor, and grey background for that field. Also went ahead and did a max-width of 800px for the the form but kept it left aligned.

**Review by pjdarch — APPROVED** [review]

**Pull request merged**